tests/NuGet.Services.AzureSearch.Tests/SearchDocumentBuilderFacts.cs

```diff
@@ -493,6 +493,7 @@ public async Task SetsExpectedProperties(SearchFilters searchFilters, string exp
     {
       ""@search.action"": ""upload"",
       ""totalDownloadCount"": 1001,
+      ""logBase2DownloadCount"": 9.9672262588359928,
```

Comment:
This looks like it can be a source of test flakiness:
* It assumes that log2 function will always produce certain value (according to [Wolfram Alpha](https://www.wolframalpha.com/input/?i=log2(1001)) the last 2 digits here are wrong);
* It assumes the JSON serialization uses certain precision and rounding rules.

If any of the above changes (with version upgrade), test might start to fail.

Reply:
Let's reassess this if/when this test proves to be flaky.